example/main.cpp

```diff
@@ -3,7 +3,7 @@
 #include "spdlog/sinks/stdout_color_sinks.h"
 #include "spdlog/sinks/basic_file_sink.h"
 
-#include "inexor_application.hpp"
+#include "../src/inexor_application.hpp"
```

Comment:
Relative imports are evil!
But this can be delayed, but we should mention that in the issue, so we dont forget it